### PR TITLE
clarify Compose install prerequisites

### PR DESCRIPTION
**Pull request by londoncalling**
### What's changed

- Re-organized topic to highlight that Docker Engine (server) is a prerequisite for Compose install
- Provided more useful links
- Changed from numbered lists to sections that call out Mac and Windows installs vs Linux

### Related

#4126 

### Direct link to updated topic in Netlify preview

https://deploy-preview-4130--docker-docs-vnext-engine.netlify.com/compose/install/

### Reviewers fyi

@mstanleyjones @barrystaes @shin- @dnephin 

Signed-off-by: Victoria Binalas <victoria.bialas@docker.com>

### CODE REVIEW

File: compose/install.md [hunk 1]
```diff
@@ -4,37 +4,52 @@ keywords: compose, orchestration, install, installation, docker, documentation
...
 ---
 
-You can run Compose on macOS, Windows and 64-bit Linux. To install it, you'll need to install Docker first.
+You can run Compose on macOS, Windows and 64-bit Linux.
 
-To install Compose, do the following:
+## Prerequisites
 
-1.  Install Docker Engine:
+You need to install Docker first, before you install and run Docker Compose.
```
Comment (shin-) [inline compose/install.md]: Technically that is not true - you can install Compose without having Docker installed first. You'll need access to a Docker engine to do anything useful with Compose, but that engine could be installed on a remote server.

File: compose/install.md [hunk 2]
```diff
@@ -4,37 +4,52 @@ keywords: compose, orchestration, install, installation, docker, documentation
...
 
-    * [Mac installation](/docker-for-mac/index.md){: target="_blank" class="_"}
+- On desktop systems like Docker for Mac and Windows, Docker Compose is
+included as part of those desktop installs. Skip down to [Install Compose on Linux systems](#install-compose-on-linux-systems).
 
-    * [Windows installation](/docker-for-windows/index.md){: target="_blank" class="_"}
+- On Linux systems, first install the
+[Docker server](/engine/installation/index.md#server){: target="_blank" class="_"}
```
Comment (shin-) [inline compose/install.md]: I think "the Docker server" should just be "Docker" here (or maybe "the Docker engine", but I think we're deprecating that term now?)

Comment (londoncalling) [inline compose/install.md]: @shin- I think we are gently trying to deprecate it so I changed it to just "Docker"

File: compose/install.md [hunk 3]
```diff
@@ -4,78 +4,92 @@ keywords: compose, orchestration, install, installation, docker, documentation
...
 
-To install Compose, do the following:
+## Prerequisites
 
-1.  Install Docker Engine:
+You need to install Docker first, before you install and run Docker Compose.
 
-    * [Mac installation](/docker-for-mac/index.md){: target="_blank" class="_"}
+- On desktop systems like Docker for Mac and Windows, Docker Compose is
+included as part of those desktop installs. Skip down to [Install Compose on Linux systems](#install-compose-on-linux-systems).
```
Comment (mstanleyjones) [inline compose/install.md]: Why would you skip down to the Linux install section if you're on a desktop?

File: compose/install.md (hunk 1 above)
Comment (londoncalling) [inline compose/install.md]: okay thanks I'll try to clarify this

File: compose/install.md [hunk 4]
```diff
@@ -4,78 +4,92 @@ keywords: compose, orchestration, install, installation, docker, documentation
...
-    [Compose repository release page on GitHub](https://github.com/docker/compose/releases){: target="_blank" class="_"}.
-    Follow the instructions from the link, which involve running the `curl` command in your terminal to download the binaries.
+## Install Compose on Linux systems
 
-    >  Got a "Permission denied" error?
-    >
-    If so, your `/usr/local/bin` directory probably isn't writable and
-    you'll need to install Compose as the superuser. Run `sudo -i`, then
-    run the download and install commands below, then `exit`.
+On **Linux**, you can download the Docker Compose binary from  the [Compose
```
Comment (mstanleyjones) [inline compose/install.md]: nit: double space

File: compose/install.md [hunk 5]
```diff
@@ -4,37 +4,53 @@ keywords: compose, orchestration, install, installation, docker, documentation
...
 
-You can run Compose on macOS, Windows and 64-bit Linux. To install it, you'll need to install Docker first.
+You can run Compose on macOS, Windows and 64-bit Linux.
 
-To install Compose, do the following:
+## Prerequisites
 
-1.  Install Docker Engine:
+Docker Compose relies on Docker Engine for any meaningful work, so make sure you
+have Docker Engine installed either locally or remote, depending on your setup.
```
Comment (londoncalling) [inline compose/install.md]: @shin- Okay I'm not sure if this is well-worded to convey the right thing, but let me know what you think.

File: compose/install.md (hunk 3 above)
Comment (londoncalling) [inline compose/install.md]: arggh! :-) okay I'll fix it.

File: compose/install.md (hunk 4 above)
Comment (londoncalling) [inline compose/install.md]: fixing

File: compose/install.md (hunk 5 above)
Comment (shin-) [inline compose/install.md]: I think that's fine :+1:

Comment (londoncalling) [inline compose/install.md]: yay! Here are some cute animal drawings.
animals

**Comment by londoncalling** [comment]
Brilliant! Thanks everyone, the topic is much better now.

**Pull request merged**